.github/workflows/build.yaml

```diff
@@ -32,12 +32,9 @@ jobs:
         with:
           images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
           tags: |
-            type=ref,event=pr
-            type=ref,event=branch,branch=main,latest=true
-            type=ref,event=branch,branch=main
-            type=ref,event=branch,branch!=main
-            type=semver,pattern={{version}},branch=main,latest=false
-            type=ref,event=tag
+            type=ref,event=pr,pattern=test
```

Comment:
what does this do? just a random non-committed tag for use inside the workflow?

Reply:
just trying to see what results I get when I change things so I know what it'll look like when we merge